src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.stories.js

```diff
@@ -30,6 +30,7 @@ Default.args = {
   title: "Digital Equity Team of San Francisco",
   description:
     "Free Fiber Initiative summary we're working together to find new and different ways of building lasting infrastructure.",
+  text: "Read More",
```

Comment:
Did you mean for this to be set to `subtitle`, and for it to actually get passed into the Article Spotlight component?

Reply:
This was a mistake